.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,30 @@
+name: Sugerir uma nova funcionalidade
+description: Utilize esse formato de issue para sugerir uma nova funcionalidade.
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: '# Sugestão de recurso'
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Descrição da sugestão
+    validations:
+      required: true
+
+  - type: textarea
+    id: steps
+    attributes:
+      label: O que deveria ser feito?
+      description: Crie etapas para esclarecer a sugestão
+    validations:
+      required: true
```

Acho que essa parte poderia ser trocada por "implementação"
```suggestion
  - type: textarea
    id: steps
    attributes:
      label: Como implementar?
      description: Tem alguma sugestão de como isso pode ser implementado?
    validations:
      required: false
```
Aí no caso seria opcional pq a pessoa talvez não tenha o conhecimento técnico e quer só dar um feedback